### ISSUE TITLE
Take server address & credentials from main Nextcloud app

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/talk-ios#12]
@marinofaggiana @Ivansss you mentioned that the apps need to be in the same namespace / by the same developer for the credentials to be shared.

Is this not something we wanted to do, moving the apps under the same account? cc @karlitschek 

Would be cool because re-typing credentials is really cumbersome. Ideally, if you already have the Nextcloud iOS app, the Nextcloud Talk iOS app should automatically log you in.

**Comment by marinofaggiana** [comment]
It is not technically possible, for now, @Ivansss do you have any idea ?

**Comment by jancborchardt** [comment]
Great, thank you @cwmke! :) 

That means to achieve this we have to move both our Nextcloud app and the Talk app into the same namespace (which they currently are not). Nextcloud is on TWS, and Nextcloud Talk through Nextcloud GmbH.

Since this enhancement would simplify the log in process _incredibly_, we should consider moving them in the same organization. cc @marinofaggiana @karlitschek @Ivansss

**Comment by mario** [comment]
Doing that atm would mean that ONE of the apps would lose current users and login credentials stored as far as I know.

Just something to keep in mind.

**Comment by nickvergessen** [comment]
Closing for now as per above?

**Comment by jancborchardt** [comment]
We need to know though that the majority of people in general do _not_ use Nextcloud yet. So while yes it will inconvenience current users of one of the apps once; the way it currently is inconveniences _all_ future possible users of both apps, which we want to be a lot of people.

TL;DR we should do this sooner than later, which is why this issue is open since 2 years already, back when we should have done it. ;)
cc @karlitschek @jospoortvliet what do you think?

**Comment by mario** [comment]
Business decision is that Talk subscription is possible without Files
subscription so I don't think this is wise. (this was communicated by Frank
a while back)

**Comment by nickvergessen** [comment]
We just had this topic last week again in the office. This is not done yet, but definitely something we want to do.

**Comment by nickvergessen** [comment]
Well it's mostlikely going to affect the other app, but anyway we don't need a stray ticket.
